### DISCUSSION TITLE
怎么离线构建指定版本kubernetes镜像呢

### DISCUSSION THREAD

**Discussion by ricardolrl** [labring/sealos#4776] [General]
如何在本地构建一个指定版本可以供sealos run使用的kubernetes镜像

**Comment by bxy4543** [comment]
sealos已经提供了较全版本的k8s镜像，可以已官方提供镜像为基础 进行相应的修改即可

**Reply by ricardolrl** [reply]
你好 目前有本地构建的需求 已在x上联系  方便的话 再次讨论 感谢

**Reply by bxy4543** [reply]
微信吧

**Reply by ricardolrl** [reply]
Ricardo_L_ 麻烦添加
